src/util/propertyTreeHelpers.ts

```diff
@@ -138,8 +138,12 @@ export function isPropertyOwnerHidden(uri: Uri, properties: Properties) {
 
 export function isSceneGraphNodeVisible(uri: Uri, properties: Properties): boolean {
   const renderableUri = sgnRenderableUri(uri);
-  const enabledValue = properties[enabledPropertyUri(renderableUri)]?.value as boolean;
-  const fadeValue = properties[fadePropertyUri(renderableUri)]?.value as number;
+  return isPropertyOwnerActive(renderableUri, properties);
+}
+
+export function isPropertyOwnerActive(uri: Uri, properties: Properties): boolean {
+  const enabledValue = properties[enabledPropertyUri(uri)]?.value as boolean;
+  const fadeValue = properties[fadePropertyUri(uri)]?.value as number;
```

Comment:
Do we need to handle if this returns undefined? What does undefined cast to number mean? 0?

Otherwise, in this file I think it would be better if the functions were defined before they were used. Here they appear in reverse order (functions use functions that are defined below them)

Reply:
Good catch!  Just copied these rows, but there is an `| undefined` missing after the `as boolean`